PWGHF/D2H/TableProducer/candidateCreatorBplusReduced.cxx

```diff
@@ -64,7 +71,7 @@ struct HfCandidateCreatorBplusReduced {
   void init(InitContext const&)
   {
     // histograms
-    registry.add("hMassBplusToD0Pi", "2-prong candidates;inv. mass (B^{+} #rightarrow #overline{D^{0}}#pi^{#plus} #rightarrow #pi^{#minus}K^{#plus}#pi^{#plus}) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{500, 3., 8.}}});
+    registry.add("hMass2BplusToD0Pi", "2-prong candidates;inv. mass (B^{+} #rightarrow #overline{D^{0}}#pi^{#plus} #rightarrow #pi^{#minus}K^{#plus}#pi^{#plus}) square (GeV^{2}/#it{c}^{4});entries", {HistType::kTH1F, {{500, 3., 8.}}});
```

Comment:
Sorry I missed that in the B0: let's fill the histogram with the mass and not the mass squared, which is tough to understand (in any case it is filled only for candidates passing the mass selection, which should not be many). The squared mass should be only useful for the data creators, where we have a lot of candidates outside the mass window (and when we repeat the selection here to have the same selection)

Reply:
Okay, I wanted to avoid using `std::sqrt` but indeed, we have less candidates after all the selections have passed.